packages/graphql/tests/integration/issues/1050.int.test.ts

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {graphql, GraphQLSchema} from "graphql";
+import {Driver, Session} from "neo4j-driver";
+import {gql} from "apollo-server";
+import neo4j from "../neo4j";
+import {getQuerySource} from "../../utils/get-query-source";
+import {generateUniqueType} from "../../utils/graphql-types";
+import {Neo4jGraphQL} from "../../../src";
+
+describe("https://github.com/neo4j/graphql/issues/1050", () => {
+    const testUser = generateUniqueType("User");
+    const testInbox = generateUniqueType("Inbox");
+    const testMessage = generateUniqueType("Message");
+    const testAttachment = generateUniqueType("Attachment");
+
+    let schema: GraphQLSchema;
+    let driver: Driver;
+    let session: Session;
+
+    beforeAll(async () => {
+        driver = await neo4j();
+
+        const typeDefs = gql`
+            type ${testUser.name} {
+                id: String
+                inboxes: [${testInbox.name}!]! @relationship(type: "OWNS", direction: OUT)
+            }
+
+            type ${testInbox.name} {
+                ownerId: String
+                messages: [${testMessage.name}!]! @relationship(type: "CONTAINS", direction: OUT)
+            }
+
+            type ${testMessage.name} {
+                ownerId: String
+                subject: String
+                body: String
```

Comment:
Just to remove some dead weight, I would try and remove anything that its not related to the test, as these unused fields

```suggestion
```